Mathlib/Logic/Function/Basic.lean

```diff
@@ -442,6 +442,8 @@ end SurjInv
 section Update
 
 variable {α : Sort u} {β : α → Sort v} {α' : Sort w} [DecidableEq α] [DecidableEq α']
+  {f g : Π a, β a} {a : α} {b : β a}
```

Comment:
```suggestion
  {f g : (a : α) → β a} {a : α} {b : β a}
```
How did this error happen? Did you not run Lean on the file before PRing?

Reply:
I don't quite remember the details, so it might be on me, but I ran it in Gitpod. Is it possible that the default Gitpod environment installs the Lean 3 extension?